doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -73,6 +73,10 @@ Matter
 
   * Support for Wi-Fi Network Diagnostic Cluster (which counts the number of packets received and transmitted on the Wi-Fi interface).
 
+* Updated:
+
+  * Enabled statically allocated heap of size 10kB, based on Zephyr `sys_heap` (:kconfig:option:`CONFIG_CHIP_MALLOC_SYS_HEAP`) to better control RAM usage of Matter applications.
```

Comment:
```suggestion
  * Control of the RAM usage of Matter applications by enabling statically allocated heap size of 10 kB, based on Zephyr's `sys_heap` (:kconfig:option:`CONFIG_CHIP_MALLOC_SYS_HEAP`).
```
Something like this, to stay consistent with "Updated: 'something by doing something'"?

Reply:
Hmm. Control of the RAM usage is the purpose, not the modification, so it completely changes the meaning. It's like prepending all actual changes with "Updated: User satisfaction by adding the support for..." :D
In which section should we put information that we enable or disable some setting?
Maybe something like "Default heap implementation to use Zephyr's `sys_heap` of size 10kB..."? Note that "to use Zephyr's sys_heap size of..." also changes the meaning since it's not size that is being modified, the whole implementation is.

Reply:
Anything that works with the structure we're using in the changelog. We simply cannot have the bullet starting with a verb if it is under a "Added"/"Updated"/"Fixed" already.

Reply:
Ok, done.